tests/Oryx.BuildImage.Tests/Python/PythonDynamicInstallationTest.cs

```diff
@@ -76,6 +76,50 @@ public void GeneratesScript_AndBuilds(string imageName)
                 result.GetDebugInfo());
         }
 
+        [Theory]
+        [InlineData("3.8.0b3")]
```

Comment:
remove 3.8.0b3 may be? and replace it with 3.8.3?

Reply:
@qianz2 3.8.3 here?

Reply:
I added it onto the other test, since this test is only testing for preview versions.